doc/whats_new.rst

```diff
@@ -148,6 +148,8 @@ BUG
 
     - Fix bug in :func:`mne.viz.plot_compare_evokeds` where ``truncate_yaxis`` was ignored (default is now ``False``), by `Jona Sassenhagen`_
 
+    - Fix bux in :func:`mne.viz.evoked._plot_evoked` where all xlabels were removed when using `spatial_colors=True`, by `Jesper Duemose Nielsen`_
```

Comment:
You need to add your webpage at the bottom. Otherwise this won't render correctly